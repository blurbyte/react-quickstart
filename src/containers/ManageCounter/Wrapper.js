import styled from 'styled-components';

const Wrapper = styled.div`
  display: inline-block;
  border-top: 1px solid #908e89;
  margin-top: 3rem;
`;

export default Wrapper;
