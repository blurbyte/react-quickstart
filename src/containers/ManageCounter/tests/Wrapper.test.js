import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Wrapper from '../Wrapper';

describe('<AppWrapper />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <Wrapper />
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
