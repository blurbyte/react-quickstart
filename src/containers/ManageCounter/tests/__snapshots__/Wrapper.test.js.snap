// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppWrapper /> should render correctly 1`] = `
.c0 {
  display: inline-block;
  border-top: 1px solid #908e89;
  margin-top: 3rem;
}

<div
  className="c0"
/>
`;
